go.mod

```diff
@@ -79,18 +79,18 @@ require (
 replace (
 	github.com/emicklei/go-restful => github.com/emicklei/go-restful v2.9.5+incompatible // keep this value in sync with k8s.io/apiserver
 	github.com/envoyproxy/go-control-plane => github.com/envoyproxy/go-control-plane v0.9.4
-	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.5.5 // keep this value in sync with k8s.io/apiserver
+	github.com/google/gnostic => github.com/google/gnostic v0.6.6 // keep this value in sync with k8s.io/apiserver
```

Comment:
This value is not in sync with https://github.com/kubernetes/apiserver/blob/7d5ba2d02807fe7c7c7dd8b43b51bd708a3ee35b/go.mod#L21
Why did you bump it?

Reply:
For the latest version, I will make it in sync.